code/controllers/subsystem/processing/quirks.dm

```diff
@@ -32,6 +34,13 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 		list("Social Anxiety", "Mute"),
 	)
 
+	/// A list of quirks that cannot be used by specific species.
+	var/static/list/species_quirk_blacklist = list(
+		SPECIES_PLASMAMAN = list(
+			"Blood Deficiency",
```

Comment:
Theres so many magic strings in here 🤢 

not your fault but sheesh

Reply:
With 515 these can all become `/datum/quirk/blood_defficiency::name`, or of course using the typepaths directly and having the serialization layer translate to strings